atlas/src/main/resources/camel/atlas_app.xml

```diff
@@ -87,6 +87,10 @@
         <constructor-arg value="atlas" />
     </bean>
 
+    <bean id="georLogger" class="org.slf4j.LoggerFactory" factory-method="getLogger" xmlns="http://www.springframework.org/schema/beans">
+        <constructor-arg value="org.georchestra.georLogger" />
```

Comment:
The main concern here is that each time the georLogger will be used, it will match the `org.georchestra.georLogger` package, which might not be intended.

Anyway, since it seems to be used only here in the XML file, that is not a big deal though. I'd have seen something like `org.georchestra.atlas.camel` which would have been more meaningfull.

Reply:
IIRC, you want me to rename the id to `org.georchestra.atlas.camel.logger` ?

Reply:
something like that, instead of georLogger